File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/UtilContentAddIpfsBody.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class UtilContentAddIpfsBody {
    /// <summary>
    /// Gets or Sets Coluuid
    /// </summary>
    [DataMember(Name="coluuid", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "coluuid")]
    public string Coluuid { get; set; }

    /// <summary>
    /// Gets or Sets Dir
    /// </summary>
    [DataMember(Name="dir", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "dir")]
    public string Dir { get; set; }

    /// <summary>
    /// Gets or Sets Filename
    /// </summary>
    [DataMember(Name="filename", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "filename")]
    public string Filename { get; set; }

    /// <summary>
    /// Gets or Sets Peers
    /// </summary>
    [DataMember(Name="peers", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "peers")]
    public List<string> Peers { get; set; }

    /// <summary>
    /// Gets or Sets Root
    /// </summary>
    [DataMember(Name="root", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "root")]
    public string Root { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class UtilContentAddIpfsBody {\n");
      sb.Append("  Coluuid: ").Append(Coluuid).Append("\n");
      sb.Append("  Dir: ").Append(Dir).Append("\n");
      sb.Append("  Filename: ").Append(Filename).Append("\n");
      sb.Append("  Peers: ").Append(Peers).Append("\n");
      sb.Append("  Root: ").Append(Root).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/util_content_add_ipfs_body.dart
part of swagger.api;

class UtilContentAddIpfsBody {
  
  String coluuid = null;

  String dir = null;

  String filename = null;

  List<String> peers = [];

  String root = null;

  UtilContentAddIpfsBody();

  @override
  String toString() {
    return 'UtilContentAddIpfsBody[coluuid=$coluuid, dir=$dir, filename=$filename, peers=$peers, root=$root, ]';
  }

  UtilContentAddIpfsBody.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    coluuid = json['coluuid'];
    dir = json['dir'];
    filename = json['filename'];
    peers = (json['peers'] as List).map((item) => item as String).toList();
    root = json['root'];
  }

  Map<String, dynamic> toJson() {
    return {
      'coluuid': coluuid,
      'dir': dir,
      'filename': filename,
      'peers': peers,
      'root': root
     };
  }

  static List<UtilContentAddIpfsBody> listFromJson(List<dynamic> json) {
    return json == null ? new List<UtilContentAddIpfsBody>() : json.map((value) => new UtilContentAddIpfsBody.fromJson(value)).toList();
  }

  static Map<String, UtilContentAddIpfsBody> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, UtilContentAddIpfsBody>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new UtilContentAddIpfsBody.fromJson(value));
    }
    return map;
  }
}
